test/zebra/payment_intents_test.clj

```diff
@@ -65,17 +67,29 @@
 
                          api-key)]
 
-    (testing "should create a valid payment intent"
-      (is (str/starts-with? (:id payment-intent) "pi_"))
-      (is (= (:object payment-intent) "payment_intent"))
-      (is (= (:status payment-intent) "requires_action"))
-      (is (= (:confirmation_method payment-intent) "automatic"))
-      (is (= (:payment_method_types payment-intent) ["card"]))
-      (is (vector? (:payment_method_types payment-intent)))
-      (is (= (:amount payment-intent) 1234))
-      (is (= (:currency payment-intent) "gbp"))
-      (is (= (:payment_method payment-intent) (:id payment-method)))
-      (is (= (:type (:next_action payment-intent)) "redirect_to_url")))))
+        (do
```

Comment:
There's no need for a `do` here. The body of a `let` is an implicit `do`.

Reply:
:+1: